src/components/Timer/Timer.jsx

```diff
@@ -0,0 +1,35 @@
+import React from 'react';
+import styles from './Timer.module.css';
+
+export class Timer extends React.Component {
+    constructor() {
+        super();
+        this.decrement = this.decrement.bind(this);
```

Comment:
Looks weird to see state definition and method binding this way round. Not wrong, just different!